keras/engine/training.py

```diff
@@ -1297,6 +1372,8 @@ def fit(self, x=None,
                 you can also pass a dictionary
                 mapping output names to Numpy arrays.
             batch_size: integer. Number of samples per gradient update.
+                Defaults to 32 if using per-sample training and no batch
```

Comment:
Maybe better to say "if training from Numpy arrays". "per-sample training" will not be immediately understandable to most users.